spec/cucumber/rake/forked_spec.rb

```diff
@@ -36,8 +36,8 @@ module Rake
       context 'when running without bundler' do
         let(:bundler) { false }
 
-        subject { Task::ForkedCucumberRunner.new(
-            libs, binary, cucumber_opts, bundler, feature_files) }
+        subject do Task::ForkedCucumberRunner.new(
```

Comment:
`do` and `end` are the starting arguments and distinct like above. I'll leave the rest out.

Reply:
All set on this one